src/language/CodeInspection.js

```diff
@@ -199,6 +199,21 @@ define(function (require, exports, module) {
     }
 
     /**
+     * Returns an array of the IDs of providers registered for a specific language
+     *
+     * @param {!string} languageId
+     * @return {Array.<string>} Names of registered providers.
```

Comment:
This should be `{?Array.<string>}`.
Or maybe it should return an empty array on the null case.